integration-test/src/main/java/com/scalar/db/api/DistributedStorageCrossPartitionScanIntegrationTestBase.java

```diff
@@ -560,7 +560,6 @@ private Scan prepareScanWithLike(boolean isLike, String pattern) {
         .table(CONDITION_TEST_TABLE)
         .all()
         .where(condition)
-        .ordering(Ordering.asc(PARTITION_KEY_NAME))
```

Comment:
I removed orderings (including at other places) to commonly use these tests in non-JDBC databases except for order-specifc tests.